Index: src/main/java/pandemic/model/ComponentsFactory.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/pandemic/model/ComponentsFactory.java	(date 1579859078000)
+++ src/main/java/pandemic/model/ComponentsFactory.java	(date 1579859078000)
@@ -248,7 +248,8 @@
         int yPos = getYCoordinate(KEY_CARD_DEFAULTPOSITION);
         String templateName = getValue(KEY_CARD_DEFAULTPOSITION, 2);
 
-		List<Integer> availableEvents = Arrays.asList(EVENTS_CORE);
+		List<Integer> availableEvents = new ArrayList<Integer>();
+		availableEvents.addAll(Arrays.asList(EVENTS_CORE));
         
         if (!useNewSpecialEvents) {
             // Classic version
